### ISSUE TITLE
support swipe to chang image like gallery

### ISSUE THREAD

**Issue by fukemy** [alantoa/react-native-lightbox#2]
as title, did u support this?

**Comment by fukemy** [comment]
yes, love u <3 , pls close this ticket as notice when it's done <3

**Comment by vbylen** [comment]
looking forward to web support 🤤

**Comment by alantoa** [comment]
@vbylen Hi!
web support I built it before, the web use [photoswipe](https://github.com/dimsemenov/photoswipe), just I'm not uploading the code yet, because currently, I'm thinking about a better API and todo planning.

you can go [here](https://showtime.xyz/@tikhonchik) to experience the lightbox of the web version.(tap `Avatar` or `Banner`)

**Comment by alantoa** [comment]
I want to do it, but currently I don't have too much time to work on that. 
Also, I'm waiting for the new implementation of Shared Element Transitions from [Reanimated](https://docs.swmansion.com/react-native-reanimated/docs/shared-element-transitions/overview/). maybe this is a better way to implement it.

**Comment by alantoa** [comment]
This package is no longer maintained. For better performance and an improved API, please consider using [galeria](https://github.com/nandorojo/galeria). It’s a great alternative for your needs.
